src/status_im/events.cljs

```diff
@@ -19,6 +19,7 @@
     status-im.contexts.emoji-picker.events
     status-im.contexts.onboarding.common.overlay.events
     status-im.contexts.onboarding.events
+    status-im.contexts.profile.edit.name.events
     status-im.contexts.profile.events
```

Comment:
I think it's better if the `profile.events` namespace require any subnamespaces within the `profile` bounded context. This way, we can try to make the `status-im.events` file only require the root of context namespaces. Sort of making each context responsible for requiring their own event namespaces in isolation. I see we are not doing this yet in other contexts, so feel free to ignore my comment.

Reply:
It makes sense, i will move all require subnamespaces in profile folder to `profile.events` from now on